pkg/meta/meta_test.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2018 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestAddOwnerReference(t *testing.T) {
```

Comment:
the implementations in the old `util` package also had unit tests for them, e.g. finalizer_test.go and references_test.go.  Should we not copy (port) those tests over to this new meta package too?

Reply:
Yeah, definitely planning to port those over. I wanted to submit this as a draft at the end of the day before I spent time on that to vet that folks liked the direction.